.github/workflows/test_published_images.yml

```diff
@@ -0,0 +1,172 @@
+name: Test published images
+
+on:
+  workflow_dispatch:
+    inputs:
+      IMAGE_VERSION:
+        required: true
+        description: A version, `5.4.1`, `latest` etc
+      EXPECTED_HZ_VERSION:
+        description: A fully-qualified version, e.g. `5.4.1`
+        required: true
+      DISTRIBUTION_TYPE:
+        required: true
+        description: The type of distribution - `oss` or `ee` - supplied as a comma-separated list (e.g. `oss`, `ee`)
+      DEFAULT_JAVA_VERSION:
+        required: true
+        description: The expected Java major version (e.g. `21`, `8`) used by default in the image
+      OTHER_JDKS:
+        required: true
+        description: The other Java variant images to test (e.g. `5-jdk21`) - supplied as a comma-separated list of major Java versions (e.g. `8, 11, 17`)
+
+env:
+  CONTAINER_NAME: my-container
+
+jobs:
+  set-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      distribution-type-matrix: ${{ steps.set-matrix.outputs.distribution-type-matrix }}
+      jdk-image-variants-matrix: ${{ steps.set-matrix.outputs.jdk-image-variants-matrix }}
+    steps:
+      - name: Parse input into matrix
+        id: set-matrix
+        run: |
+          # https://unix.stackexchange.com/a/719752 + trim
+          matrix=$(echo '"${{ inputs.DISTRIBUTION_TYPE }}"' | jq --compact-output 'split(",") | map(gsub("^\\s+|\\s+$"; ""))' )
+          echo "distribution-type-matrix=$matrix" >> $GITHUB_OUTPUT
+
+          # Add an "empty" option for base image
+          matrix=$(echo '"${{ inputs.OTHER_JDKS }}"' | jq --compact-output 'split(",") + [""] | map(gsub("^\\s+|\\s+$"; ""))' )
+          echo "jdk-image-variants-matrix=$matrix" >> $GITHUB_OUTPUT
+
+  test:
+    runs-on: ubuntu-latest
+    needs: set-matrix
```

Comment:
> understood I asked because I recall two separate jobs will appear in the PR checks section. Not a big deal though

This isn't a PR check so shouldn't be an issue.